src/android/CameraLauncher.java

```diff
@@ -547,7 +547,7 @@ private String ouputModifiedBitmap(Bitmap bitmap, Uri uri) throws IOException {
         String realPath = FileHelper.getRealPath(uri, this.cordova);
 
         // Get filename from uri
-        String fileName = realPath != null ?
+        String fileName = realPath != null && !realPath.isEmpty() ?
```

Comment:
Looks like TextUtils.isEmpty is not currently used anywhere in Cordova Android code, so I'm OK with leaving this as-is.